src/Libraries/CoreNodeModels/ColorRange.cs

```diff
@@ -20,6 +20,8 @@ namespace DSCoreNodesUI
     [NodeCategory("Core.Color.Create")]
     [NodeDescription("ColorRangeDescription",typeof(DSCoreNodesUI.Properties.Resources))]
     [NodeSearchTags("ColorRangeSearchTags", typeof(DSCoreNodesUI.Properties.Resources))]
+    [InputParameters("colors", "indices", "value")]
```

Comment:
Is there some way we can ensure that these names and the port names are in sync?